libs/ledger-live-common/src/families/nym/banner.test.ts

```diff
@@ -0,0 +1,181 @@
+import { getAccountBannerState } from "./banner";
+import * as preloadedData from "./preloadedData";
+import * as logic from "./logic";
+import type { NymAccount, NymValidatorItem } from "./types";
+import data from "./preloadedData.mock";
+import { LEDGER_VALIDATOR_ADDRESS } from "./utils";
+import { BigNumber } from "bignumber.js";
+
+jest.mock("./js-prepareTransaction", () => ({
+  calculateFees: jest.fn(() => Promise.resolve({})),
+}));
+
+const ledgerValidator: NymValidatorItem | undefined = data.validators.find(
+  (x) => x.validatorAddress === LEDGER_VALIDATOR_ADDRESS
+);
+const expensiveValidator: NymValidatorItem | undefined = data.validators.find(
+  (x) =>
+    x.validatorAddress === "nymvaloper1qs8tnw2t8l6amtzvdemnnsq9dzk0ag0z52uzay"
```

Comment:
```suggestion
    x.validatorAddress === "nvaloper1qs8tnw2t8l6amtzvdemnnsq9dzk0ag0z52uzay"
```

The prefix is just `n`